gadgets/snapshot_socket/README.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: snapshot_socket
+---
```

Comment:
Please add a sidebar_position as done in https://github.com/inspektor-gadget/inspektor-gadget/commit/2e64540e15a111e92feb89d6140a0546de85b573 .